modules/nodes-cma-autoscaling-custom-prometheus-config.adoc

```diff
@@ -70,7 +94,7 @@ Labels:              <none>
 Annotations:         <none>
 Image pull secrets:  thanos-dockercfg-nnwgj
 Mountable secrets:   thanos-dockercfg-nnwgj
-Tokens:              thanos-token-9g4n5 <1>
+Tokens:              thanos-token <1>
 Events:              <none>
 
 ----
```

Comment:
Line 116 and 119, both mention `thanos-token-9g4n5`, that should be `thanos-token` now, because the name was defined by the new secret we are now creating above.
```
  secretTargetRef: <1>
  - parameter: bearerToken <2>
    name: thanos-token-9g4n5 <3>
    key: token <4>
  - parameter: ca
    name: thanos-token-9g4n5
    key: ca.crt
```

Reply:
outdated